data/hai/hai ships.txt

```diff
@@ -733,6 +804,69 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 		`"Bufaer" Atomic Steering`
 		"Hyperdrive"
 
+ship "Shield Beetle Prototype"
+	sprite "ship/hai shield beetle prototype"
+	thumbnail "thumbnail/hai shield beetle"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Unfettered Militia"
+		"cost" 25400000
+		"shields" 42000
```

Comment:
```suggestion
		"shields" 40600
```

TBF, even with the other changes, this still seems just a -tad- too sturdy.

I'm kind of thinking this should come earlier, as an explicitly defensively reinforced version (with only 36400 shields though, as still a large improvement over SB), and where you get this now should be a Solifuge version which is explicitly a prototype for the upcoming war with similar reductions (maybe 5 turrets instead of 7, and/or only 4 or 5 fighter bays, but with enough weapon/outfit space for two of the pentapulser turrets [which you also otherwise can't get yet] and 43200 shields and 12300 hull).